examples/datasets/opm_data.py

```diff
@@ -93,7 +93,7 @@
 
 # Fit dipoles on a subset of time points
 with mne.use_coil_def(coil_def_fname):
-    dip_opm, _ = mne.fit_dipole(evoked.copy().crop(0.015, 0.080),
+    dip_opm, _ = mne.fit_dipole(evoked.copy().crop(0.040, 0.080),
```

Comment:
why did you change this?

Reply:
To make it run a little bit faster. The earlier outputs aren't really used anyway